package.json

```diff
@@ -31,6 +31,7 @@
     "webpack": "2.0.6-beta"
   },
   "scripts": {
-    "build": "webpack"
+    "build": "webpack",
+    "watch-build": "webpack --watch --progress"
```

Comment:
I like the convention of using a colon to "scope" related scripts together. Here, that would be ``build:watch`` since it's a specific kind of build script.

I'm just mentioning this as a very general code quality type of thing. ``watch-build`` is fine too, just let me know what you would like to do :-D.